test/service.go

```diff
@@ -65,9 +65,41 @@ func CreateService(ctx *Context, name, namespace, image string) (*servingv1beta1
 	return service, nil
 }
 
+func GetPod(ctx *Context, ns string) {
```

Comment:
We should change the name of function `GetPod()` . The name sounds like similar function `GetConfiguration()` but it does not actually.

Reply:
Yeah let's make this `WaitForControllerEnvironment` and pass in the key/value you want to see. You're going to need the inverse once we can enable the disabling test.